package.json

```diff
@@ -5,12 +5,16 @@
   "dependencies": {
     "express": "^4.17.1",
     "node": "^10.15.3",
+    "nodemon": "^1.19.4",
+    "npm-run-all": "^4.1.5",
     "react": "^16.11.0",
     "react-dom": "^16.11.0",
     "react-router-dom": "^5.1.2",
     "react-scripts": "3.2.0"
   },
   "scripts": {
+    "start-servers": "npm-run-all -p start start-back",
+    "start-back": "nodemon src/backend/app.js",
     "start": "react-scripts start",
```

Comment:
Can you please rename `start` with `start-front`, for consistency reasons?

Reply:
Thank you, will do